working/wildcards/feature-specification.md

```diff
@@ -218,8 +219,18 @@ class C {
 }
 ```
 
+It is a compile-time error if a named initializing formal has the name `_`:
+
+```dart
+class C {
+  var _;
+  
+  C({this._}); // Error.
+}
+```
+
 But no *parameter* with that name is bound, which means `_` can't be accessed
-inside the initializer list. In the body is fine, since that refers to the
+inside the initializer list. In the body it is fine, since that refers to the
```

Comment:
Can we rewrite "In the body it is fine" to something like "Using `_` in the body is fine," or something similar. I can't really tell what "it" is and reading that phrase is somewhat difficult.

Reply:
Good point, done!